lib/active_fedora/fixity_service.rb

```diff
@@ -32,10 +32,7 @@ def fixity_graph
       ::RDF::Graph.new << ::RDF::Reader.for(:ttl).new(response.body)
     end
 
-    def status_url
-      ::RDF::URI("http://fedora.info/definitions/v4/repository#status")
```

Comment:
Why not leave this method and make it return `ActiveFedora::RDF::Fcrepo4.status`?  That way it's more easily overridable.